swagger.yaml

```diff
@@ -198,6 +198,12 @@ parameters:
     description: "Filter resource version, as returned by a previous ETag, to be validated; or '*' to skip the version check"
     in: header
     type: string
+  is-based-on:
```

Comment:
We have used camel_case for query parameter previously; not sure if we intend in changing this?

Reply:
Should now be amended